packages/dialtone-vue2/components/split_button/split_button.vue

```diff
@@ -0,0 +1,282 @@
+      type: String,
+      default: 'left',
+      validator: (position) => Object.keys(ICON_POSITION_MODIFIERS).includes(position),
+    },
+
+    /**
+     * Used to customize the alpha label container
+     */
+    alphaLabelClass: {
+      type: [String, Array, Object],
+      default: '',
+    },
+
+    /**
+     * Whether the alpha button should display a loading animation or not.
+     * @values true, false
+     */
+    alphaLoading: {
+      type: Boolean,
+      default: false,
+    },
+
+    /**
+     * Text shown in tooltip when you hover the alpha button
+     */
+    alphaTooltipText: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * Determines whether a screenreader reads live updates of
+     * the button content to the user while the button
+     * is in focus. default is to not.
+     * @values true, false
+     */
+    assertiveOnFocus: {
+      type: Boolean,
+      default: false,
+    },
+
+    /**
+     * HTML button disabled attribute
+     * <a
+     *   class="d-link"
+     *   href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#disabled"
+     *   target="_blank"
+     * >
+     *   (Reference)
+     * </a>
+     * @values true, false
+     */
+    disabled: {
+      type: Boolean,
+      default: false,
+    },
+
+    /**
+     * The fill and outline of the button associated with its visual importance.
+     * @values clear, outlined, primary
+     */
+    importance: {
+      type: String,
+      default: 'primary',
+      validator: (i) => Object.keys(BUTTON_IMPORTANCE_MODIFIERS).includes(i),
+    },
+
+    /**
+     * The color of the button.
+     * @values default, muted, danger, inverted
+     */
+    kind: {
+      type: String,
+      default: 'default',
+      validator: (k) => Object.keys(BUTTON_KIND_MODIFIERS).includes(k),
+    },
+
+    /**
+     * Determines whether the omega button should have active styling
+     * default is false.
+     * @values true, false
+     */
+    omegaActive: {
+      type: Boolean,
+      default: false,
+    },
+
+    /**
+     * Descriptive label for the omega button
+     */
+    omegaAriaLabel: {
+      type: String,
+      default: null,
+    },
+
+    /**
+     * Element ID, useful in case you need to reference the button
+     * as an external anchor for popover
+     */
+    omegaId: {
+      type: String,
+      default: undefined,
+    },
+
+    /**
+     * Text shown in tooltip when you hover the omega button
+     */
+    omegaTooltipText: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * The size of the button.
+     * @values xs, sm, md, lg, xl
+     */
+    size: {
+      type: String,
+      default: 'md',
+      validator: (s) => Object.keys(BUTTON_SIZE_MODIFIERS).includes(s),
+    },
+
+    /**
+     * Button width, accepts
+     * <a class="d-link" href="https://developer.mozilla.org/en-US/docs/Web/CSS/width" target="_blank">
+     *   CSS width attribute
+     * </a>
+     * values
+     */
```

Comment:
not sure the `<a>` is necessary, could be just `accepts CSS width attribute.`

Reply:
That <a> makes this actually a link

<img width="838" alt="Screenshot 2024-07-30 at 1 53 38 p m" src="https://github.com/user-attachments/assets/8e5ffb75-226d-40a4-8991-dbdf83abfe25">

Reply:
@julio does it show as a link if you use `@see`? not sure if storybook supports that, but it is the valid jsdoc syntax for a reference link

Reply:
Ya I know it was for it, we made some of them, but we will drop storybook soon.

Reply:
> @julio does it show as a link if you use `@see`? not sure if storybook supports that, but it is the valid jsdoc syntax for a reference link

It doesn't work for storybook unfortunately.

> Ya I know it was for it, we made some of them, but we will drop storybook soon.

We're not actually dropping storybook, we're only merging the docs so they're in one place only. Storybook is a great place to play with the components, even if we have the combinator we'll need storybook to continue doing visual testing at least. 

> But beyond the usage on storybook, I saw it unnecessary for a width or disabled attribute.

I don't see how this hurts in any way, any additional help that you can provide is great, I know that those attributes are "simple" but those links provide more information that we don't need to include in our docs, so it makes sense to reference the links where someone can find more details about it quickly.

Reply:
> It doesn't work for storybook unfortunately.

k no prob, we can use the current method then. I think I remember having this exact conversation before anyways.